### PR TITLE
[TA2388] quiescing sync IO during snapshot command

### PR DESCRIPTION
**Pull request by mynktl**
Changes : 
- quiescing sync IO during snapshot command
- Changing ISTGT logging from syslog to fprintf (stderr)
- Changing rebuild wait time for the newly added replica to 2 times of replica_timeout
- Fixing deadlock in update_replica_entry

### CODE REVIEW

**Review by vishnuitta — COMMENTED** [review]
Changes are good.
- Instead of using INCREASE../DECREASE.. kind of macros, can we change them to INCREMENT../DECREMENT..?
- Lets leave build_rcmd as it is, and leave to its callers to update the inflight count
- automated test cases are required

File: src/replication_log.h [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	LOG_LEVEL_INFO,
+	LOG_LEVEL_DEBUG,
+	LOG_LEVEL_MAX,
+};
+
+int replication_log_level;
+
+#define REPLICA_LOG(fmt, ...)						\
+{									\
+	if (replication_log_level <= LOG_LEVEL_INFO)			\
```
Comment (vishnuitta) [inline src/replication_log.h]: these conditions should be reverse if I'm not wrong

Comment (mynktl) [reply src/replication_log.h]: yes

**Review by vishnuitta — APPROVED** [review]
changes are good

**Pull request merged**